src/start.js

```diff
@@ -11,3 +11,11 @@ export function start() {
 export function isStarted() {
 	return started;
 }
+
+const startWarningDelay = 3000;
```

Comment:
Is 3 seconds long enough? I think it probably is, but maybe 5 seconds would be better?

Also assuming that I have a use case with a long delay before I bootstrap any child applications. Say a really simple login page or something. It might be nice to suppress the warning; but I imagine that is probably over-architecting.